Plugins/ScenarioLoader/Plugin.cs

```diff
@@ -55,29 +60,67 @@ private async Task ExecuteFileAsync(IServer server, string path)
                 switch (parts[0].ToUpper())
                 {
                     case "SPAWN":
-                        Console.WriteLine("Creating aircraft: " + parts[1]);
-                        IAircraft? aircraft = server.SpawnAircraft(
-                            parts[1],
-                            new('I', 'S', "1/A320/M-SDG/LB1", "N450", "LJLJ", new(), new(), "F320", "LJMB", 0, 0, 0, 0, "????", "RMK/PLUGIN GENERATED AIRCRAFT. FLIGHT PLAN MAY BE INACCURATE.", "DCT"),
-                            new() { Latitude = double.Parse(parts[3]), Longitude = double.Parse(parts[4]) },
-                            float.Parse(parts[6]),
-                            uint.Parse(parts[8]),
-                            int.Parse(parts[10]) * 100
-                        );
-
-                        if (aircraft is not null)
-                            _aircraft.Add(parts[1], aircraft);
 
+                        if (parts.Length > 11)
+                        {
+                            string joined = string.Join(' ', parts);
+
+                            string[] split_flight = joined.Split(';');
+
+                            foreach (string str in split_flight) {
+
+                                Console.WriteLine(str + '\n');
+                            }
```

Comment:
```suggestion
                            foreach (string str in split_flight)
                                Console.WriteLine(str + '\n');
```
For style consistency, either remove the braces or put the opening brace on its own line.

Reply:
Removed braces for consistency.